.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.c ident
```

Comment:
I did not know about this feature and always wondered what these things `(/* $Id$ */`) are in the code?
Excuse my ignorance, what is the benefit of this feature?

It might be a relic of the good old CVS https://stackoverflow.com/questions/2763070/purpose-of-id-line-in-the-header-of-source-files

I thought about submitting a pull request to remove all these things :smile:

Reply:
`/* $Id$ */` is crap left from CVS days; I would rather remove those :)